.github/workflows/main.yml

```diff
@@ -37,12 +37,22 @@ jobs:
         echo Using Bufbuild version $BUFVERSION
         curl -sSL https://github.com/bufbuild/buf/releases/download/$BUFVERSION/buf-$(uname -s)-$(uname -m) > buf
         chmod a+x buf
+
+    - name: Download Buildifier
+      run: |
+        BUILDIFIER="3.4.0"
+        echo Using Buildifier version $BUILDIFIER
+        curl -sSLO https://github.com/bazelbuild/buildtools/releases/download/$BUILDIFIER/buildifier > buildifier
+        chmod a+x buildifier
         
     - name: Kotlin lint check
       run: ./ktlint --android domain/src/**/*.kt utility/src/**/*.kt data/src/**/*.kt app/src/**/*.kt testing/src/**/*.kt && echo "Lint completed successfully"
     
     -  name: Protobuf lint check
        run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
+
+    - name: Bazle lint check
+      run: ./buildifier --lint=warn --mode=check --warnings=all BUILD.bazel WORKSPACE oppia_android_test.bzl && echo "Bazle lint check completed successfully"
```

Comment:
Can we run this on all bzl, bazel, and WORKSPACE files?

Reply:
Yes, it covers all. 
Also we one file in this PR too - oppia_android_test.bzl